lib/ui/window.dart

```diff
@@ -666,6 +666,24 @@ class Window {
   WindowPadding get viewPadding => _viewPadding;
   WindowPadding _viewPadding = WindowPadding.zero;
 
+  /// The number of physical pixels on each side of the display rectangle into
+  /// which the application can render, but over which the operating system
+  /// will likely place system gestures.
```

Comment:
but over which the operating system will likely place system gestures => but where the operating system will consume input gestures for the sake of system navigation.